contentcuration/contentcuration/migrations/0114_contentnode_complete.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2020-08-28 12:49
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contentcuration', '0113_channel_tagline'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='contentnode',
+            name='complete',
+            field=models.BooleanField(default=True),
```

Comment:
In the past we've set fields to null and then applied a default later for migrations, is this not necessary here? (I don't know - genuinely curious!)

Reply:
I tried setting it, but apparently it doesn't work for BooleanFields